code/game/objects/items/devices/transfer_valve.dm

```diff
@@ -206,17 +215,12 @@
 		log_game("Bomb valve opened at [A.name] ([bombturf.x],[bombturf.y],[bombturf.z]) with [attached_device ? attached_device : "no device"], attached by [attacher_name]. Last touched by: [key_name(mob)]")
 		merge_gases()
 		spawn(20) // In case one tank bursts
-			for(var/i=0,i<5,i++)
-				src.update_icon()
+			for(var/i in 1 to 5)
+				update_icon()
 				sleep(10)
-			src.update_icon()
+			update_icon()
 
-	else if(valve_open==1 && (tank_one && tank_two))
+	else if(valve_open && tank_one && tank_two)
 		split_gases()
 		valve_open = 0
-		src.update_icon()
-
-// this doesn't do anything but the timer etc. expects it to be here
-// eventually maybe have it update icon to show state (timer, prox etc.) like old bombs
-/obj/item/device/transfer_valve/proc/c_state()

```

Comment:
https://github.com/ParadiseSS13/Paradise/pull/6349/files#diff-dcb33c6ba2f9f375aec134af5ca806a1R195

This is the only known comment ExaDev ever left.